roles/preflight-checks/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+skip_prechecks_all: false
```

Comment:
Agree, use appropriate tag names at the site.yml level for preflight itself, and then more specific tags inside the role to be able to skip/target specific parts.